tests/DataObjectTest.py

```diff
@@ -110,3 +114,19 @@ def __init__(self, a=None, **kw):
     def test_rdfs_comment_property(self):
         a = DataObject(rdfs_comment='Hello')
         self.assertIn('Hello', a.rdfs_comment())
+
+    def test_contextualize_class_augment(self):
+        ctx = Mock()
+        ctx.identifier = None
+        a = DataObject(rdfs_comment='Hello')
+        res = a.contextualize_class_augment(ctx)
```

Comment:
It is the class which has `contextualize_class_augment`, not the instance.

Reply:
But is not a classmethod ?